lighthouse-core/lib/rect-helpers.js

```diff
@@ -246,3 +248,9 @@ module.exports = {
   filterOutRectsContainedByOthers,
   filterOutTinyRects,
 };
+
+if (typeof module !== 'undefined' && module.exports) {
+  module.exports = RectHelpers;
+} else {
+  self.RectHelpers = RectHelpers;
```

Comment:
is this only for `getRectCenterPoint`? *Strongly* prefer just duplicating it in `element-screenshot-renderer.js` in that case. Sharing code between the report renderer and the node code is messy, confuses tsc and requires a bunch of jsdom juggling. It's not a big deal to have two copies of it.